packages/application-shell/src/components/application-shell/redirect-to-login.tsx

```diff
@@ -29,6 +31,9 @@ const generateAndCacheNonceWithState = (state: AuthorizeSessionState) => {
 
 const RedirectToLogin = () => {
   const location = useLocation();
+  const servedByProxy = useApplicationContext(
```

Comment:
Nit: we use this pattern four times already with this being the fifth. Maybe we can extract it now into a `useIsServerdByProxy` (in this or a follow up PR). As it's quite impactful logic resulting from this it might be good not to duplicate this too much even if it's simple.